cypress/mockApis/data.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
```

Comment:
maybe this could go somewhere better but probably fine for now (could even look at sharing these test data helpers across unit and integration tests?)